pkg/virt-controller/watch/vm/vm.go

```diff
@@ -517,6 +517,14 @@ func (c *Controller) handleDataVolumes(vm *virtv1.VirtualMachine) (bool, error)
 		} else if curDataVolume.Status.Phase != cdiv1.Succeeded &&
 			curDataVolume.Status.Phase != cdiv1.WaitForFirstConsumer &&
 			curDataVolume.Status.Phase != cdiv1.PendingPopulation {
+
+			if curDataVolume.Status.Phase == cdiv1.Pending {
+				err := storagetypes.HasDataVolumeExceededQuota(vm.Namespace, vm.Spec.Template.Spec.Volumes, c.dataVolumeStore)
```

Comment:
why do you need to go over all the other volumes when you already are working on a specific dv?

Reply:
Right, this is a leftover from the `setPrintableStatus()` usage of the same func, which was removed. Fixing.